plugin/pom.xml

```
@@ -44,6 +44,10 @@
         <no-test-jar>false</no-test-jar>
         <node.version>18.15.0</node.version>
         <yarn.version>1.22.19</yarn.version>
+
+        <!-- Opt in to Jenkins handling eslint results -->
+        <maven.test.failure.ignore>false</maven.test.failure.ignore>
+        <testFailureIgnore>${maven.test.failure.ignore}</testFailureIgnore>
```

Should this be in `plugin-pom`?

Safer to configure this in `build/plugins/*/configuration` than to rely on setting a generic-sounding property.

---

This is for the opting in to say that that we have configured test/lint properly for Jenkins build reporting. It is in `plugin-pom` here: https://github.com/jenkinsci/plugin-pom/pull/1016/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R149

We could prefix it, e.g. something like:
```suggestion
        <maven.test.failure.ignore>false</maven.test.failure.ignore>
        <frontend.testFailureIgnore>${maven.test.failure.ignore}</frontend.testFailureIgnore>
```

---

Yes I think prefixing it would be prudent. As written it looks like it is a magic property name interpreted by some unknown mojo. But https://github.com/eirslett/frontend-maven-plugin?tab=readme-ov-file#ignoring-failure seems to imply that regular `maven.test.failure.ignore` is honored? https://github.com/eirslett/frontend-maven-plugin/blob/4a501a10677716a0e6676b603e3b93bc6bf319f1/frontend-maven-plugin/src/main/java/com/github/eirslett/maven/plugins/frontend/mojo/AbstractFrontendMojo.java#L33-L40 so why do we even need to configure the plugin specially for this?

---

see https://github.com/jenkinsci/plugin-pom/pull/1015#issuecomment-2368535480